management/Makefile

```diff
@@ -1,13 +1,13 @@
 
 # The kname of the context for the management cluster
 # The name of the context for your Kubeflow cluster
-NAME=$(shell yq r ./instance/settings.yaml name)
+NAME=$(shell yq -r .name ./instance/settings.yaml)
```

Comment:
Which version of yq are you using? I'm using

yq --version
yq version 2.4.0

File goes ahead of the predicate

```
yq
Usage:
  yq [flags]
  yq [command]

Available Commands:
  delete      yq d [--inplace/-i] [--doc/-d index] sample.yaml a.b.c
  help        Help about any command
  merge       yq m [--inplace/-i] [--doc/-d index] [--overwrite/-x] [--append/-a] sample.yaml sample2.yaml
  new         yq n [--script/-s script_file] a.b.c newValue
  prefix      yq p [--inplace/-i] [--doc/-d index] sample.yaml a.b.c
  read        yq r [--doc/-d index] sample.yaml a.b.c
  write       yq w [--inplace/-i] [--script/-s script_file] [--doc/-d index] sample.yaml a.b.c newValue

Flags:
  -h, --help      help for yq
  -t, --trim      trim yaml output (default true)
  -v, --verbose   verbose mode
  -V, --version   Print version information and quit

Use "yq [command] --help" for more information about a command.

```

Reply:
yq 2.7.2
usage: yq [options] <jq filter> [YAML file...]
          jq_filter [files [files ...]]

Some of the options include:
  -c               compact instead of pretty-printed output;
  -n               use `null` as the single input value;
  -e               set the exit status code based on the output;
  -s               read (slurp) all inputs into an array; apply filter to it;
  -r               output raw strings, not JSON texts;

Similar examples here
https://kislyuk.github.io/yq/
`yq .foo.bar input.yml`

Reply:
I will match my version to yours and close the PR. Also please see line 23 if `set-values:` is intentional